plugins/zlux-agent/doc/swagger/server-plugins-api.yaml

```diff
@@ -0,0 +1,784 @@
+#  This program and the accompanying materials are
+#  made available under the terms of the Eclipse Public License v2.0 which accompanies
+#  this distribution, and is available at https://www.eclipse.org/legal/epl-v20.html
+#  
+#  SPDX-License-Identifier: EPL-2.0
+#  
+#  Copyright Contributors to the Zowe Project.
+
+swagger: "2.0"
+info:
+  description: "These are the Swagger documents on dynamic app server configuration for the Zowe desktop, used to control the plugin and services lifecycle. You can find more documentation about Zowe at [https://docs.zowe.org/stable/]."
+  version: "1.0.0"
+  title: "Zowe plugin and services lifecycle"
+  license:
+    name: "EPL-2.0"
+host: "localhost"
+basePath: /
+tags:
+- name: "server"
+  description: "Server essentials"
+  externalDocs:
+    description: "Find out more"
+    url: "https://github.com/zowe/zlux-app-server"
+- name: "plugins"
+  description: "Plugin lifecycle changes"
+
+schemes:
+- "https"
+- "http"
+paths:
+
+  /server/:
+    get:
+      tags:
+      - "server"
+      summary: "Retrieves list of accessible server endpoints"
+      description: "Retrieves list of accessible server endpoints"
+      produces:
+      - "application/json"
+      responses:
+        200:
+          description: "Success"
+          schema:
+            $ref: "#/definitions/serverLinksDefinition"
+        401:
+          description: "Unauthorized"
+        506:
+          description: "Set dataserviceAuthentication.rbac to true in server configuration"
+          
+  /server/agent:
+    get:
+      tags:
+      - "server"
+      summary: "Get the agent server (often ZSS server)"
+      description: "Get the agent server (often ZSS server)"
+      produces:
+      - "application/json"
+      responses:
+        200:
+          description: "Success"
+        401:
+          description: "Unauthorized"
+        506:
+          description: "Set dataserviceAuthentication.rbac to true in server configuration"
+          
+  /server/agent/config:
+    get:
+      tags:
+      - "server"
+      summary: "Get the config of the agent"
+      description: "Get the config of the agent"
+
+      produces:
+      - "application/json"
+      responses:
+        200:
+          description: "Success"
+          schema:
+            $ref: "#/definitions/agentConfigDefinition"
+        401:
+          description: "Unauthorized"
+        506:
+          description: "Set dataserviceAuthentication.rbac to true in server configuration"
+          
+  /server/agent/log:
```

Comment:
example of a route thats perfectly valid on zss, because its the agent.

Reply:
yes, separated into separate swagger for agent &server